spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 require 'active_storage_validations/matchers'
 require 'webmock/rspec'
 
+if Rails.env.test?
```

Comment:
No need to check this inside the `spec_helper`. This file only gets loaded in testing already.